espanol/pensamiento/guerra.md

```diff
@@ -5,4 +5,5 @@ Ser
 
 Tras terminar con sus lamentaciones:
 
-[Una reflexi�n de gran importancia le asalt�](estoicismo/estoicismo.md)
\ No newline at end of file
+[Una reflexi�n de gran importancia le asalt�](estoicismo/estoicismo.md)
+[Se sentó en el suelo y se puso a meditar](meditacion/meditar.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.